Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -241,6 +241,14 @@ static void softlinkDataDetectorsFrameworks()
 
 void WebProcess::platformInitializeWebProcess(WebProcessCreationParameters& parameters)
 {
+    // Log a long message to make sure the XPC connection to the log daemon for oversized messages is opened.
+    // This is needed to block launchd after the WebContent process has launched, since access to launchd is
+    // required when opening new XPC connections.
+    char stringWithSpaces[1024];
+    memset(stringWithSpaces, ' ', sizeof(stringWithSpaces));
+    stringWithSpaces[sizeof(stringWithSpaces) - 1] = 0;
+    WEBPROCESS_RELEASE_LOG(Process, "%p WebProcess::platformInitializeWebProcess %s", this, stringWithSpaces);
+
```

Comment:
I prefer a helper function because a helper function encapsulates the mechanism for initializing the log daemon. Especially nice in cases like this, where the implementation is a little surprising, and might change once we have explicit SPI for this.

Something like "void initializeXPCConnectionToLogd()".

Reply:
> I'd suggest a helper function here because a helper function encapsulates the mechanism for initializing the log daemon. Encapsulation is always valuable for reducing complexity, and it is especially nice in cases like this, where the implementation is a bit surprising, and subject to change once we have explicit SPI for this.
> 
> Something like "void initializeXPCConnectionToLogd()".

Ah, yes, that is a good point! Fixed in latest patch.

Thanks for reviewing!